src/persistence/db/company.rs

```diff
@@ -133,6 +133,8 @@ pub struct CompanyKeysDb {
     pub id: Option<Thing>,
     pub public_key: String,
     pub private_key: String,
+    pub rsa_public_key: String,
+    pub rsa_private_key: String,
```

Comment:
Who are the actors that have this state and therefor access to this private key. Can this be multiple company members or just the creator of the company? I am not completely clear on this company structure.